doc/release-notes/template.sgml

```diff
@@ -0,0 +1,117 @@
+<!doctype linuxdoc system>
+<article>
+<title>Squid @SQUID_VERSION@ release notes</title>
+<author>Squid Developers</author>
+
+<abstract>
+This document contains the release notes for version @SQUID_RELEASE@ of Squid.
+Squid is a WWW Cache application developed by the National Laboratory
+for Applied Network Research and members of the Web Caching community.
+</abstract>
+
+<toc>
+
+<sect>Notice
+<p>The Squid Team are pleased to announce the release of Squid-@PACKAGE_VERSION@ for testing.
+
+This new release is available for download from <url url="http://www.squid-cache.org/Versions/v@SQUID_RELEASE@/"> or the
+ <url url="http://www.squid-cache.org/Download/http-mirrors.html" name="mirrors">.
+
+<p>While this release is not deemed ready for production use, we believe it is ready for wider testing by the community.
+
+<p>We welcome feedback and bug reports. If you find a bug, please see <url url="http://wiki.squid-cache.org/SquidFaq/BugReporting">
+   for how to submit a report with a stack trace.
+
+<sect1>Known issues
+<p>Although this release is deemed good enough for use in many setups, please note the existence of
+<url url="http://bugs.squid-cache.org/buglist.cgi?query_format=advanced&amp;product=Squid&amp;bug_status=UNCONFIRMED&amp;bug_status=NEW&amp;bug_status=ASSIGNED&amp;bug_status=REOPENED&amp;version=@SQUID_RELEASE@" name="open bugs against Squid-@SQUID_RELEASE@">.
+
+<sect1>Changes since earlier releases of Squid-@SQUID_RELEASE@
+<p>
+The Squid-@SQUID_RELEASE@ change history can be <url url="https://github.com/squid-cache/squid/commits/v@SQUID_RELEASE@" name="viewed here">.
+
+
+<sect>Major new features since Squid-@SQUID_RELEASE_OLD@
+<p>Squid-@SQUID_RELEASE@ represents a new feature release above Squid-@SQUID_RELEASE_OLD@.
```

Comment:
FYI: These `@SQUID_RELEASE_OLD@` are intended for use by the (currently future) series release script to update when it automatically generates a new series `release-N.sgml.in` from this template.

Reply:
As of [9c6d0c6](https://github.com/squid-cache/squid/pull/1264/commits/9c6d0c658b56041b5ea0283feb5139eb649e4262) that replacement is now done here too.